contrib/gitian-keys/keys.txt

```diff
@@ -1,41 +1,51 @@
 9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C Aaron Clauson (sipsorcery)
 617C90010B3BD370B0AC7D424BB42E31C79111B8 Akira Takizawa
 E944AE667CF960B1004BC32FCA662BE18B877A60 Andreas Schildbach
-152812300785C96444D3334D17565732E08E5E41 Andrew Chow
+152812300785C96444D3334D17565732E08E5E41 Andrew Chow (achow101)
 590B7292695AFFA5B672CBB2E13FC145CD3F4304 Antoine Poinsot (darosior)
 0AD83877C1F0CD1EE9BD660AD7CC770B81FD22A8 Ben Carman (benthecarman)
 912FD3228387123DC97E0E57D5566241A0295FA9 BtcDrak
 C519EBCF3B926298946783EFF6430754120EC2F4 Christian Decker (cdecker)
+18AE2F798E0D239755DA4FD24B79F986CBDF8736 Chun Kuan Le (ken2812221)
 F20F56EF6A067F70E8A5C99FFF95FAA971697405 centaur
-C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields
+C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields (cfields)
 BF6273FAEF7CC0BA1F562E50989F6B3048A116B5 Dev Random
-6D3170C1DC2C6FD0AEEBCA6743811D1A26623924 Douglas Roark
+6D3170C1DC2C6FD0AEEBCA6743811D1A26623924 Douglas Roark (droark)
+1C6621605EC50319C463D56C7F81D87985D61612 Emanuele Cisbani (cisba)
 9A1689B60D1B3CCE9262307A2F40A9BF167FBA47 Erik Mossberg (erkmos)
 D35176BE9264832E4ACA8986BF0792FBE95DC863 fivepiece
+6F993B250557E7B016ADE5713BDCDA2D87A881D9 Fuzzbawls (Fuzzbawls)
 01CDF4627A3B88AAE4A571C87588242FBE38D3A8 Gavin Andresen
 D1DBF2C4B96F2DEBF4C16654410108112E7EA81F Hennadii Stepanov (hebasto)
+A2FD494D0021AA9B4FA58F759102B7AE654A4A5A Ilyas Ridhuan (IlyasRidhuan)
+D3F22A3A4C366C2DCB66D3722DA9C5A7FA81EA35 Jarol Rodriguez (jarolrod)
+7480909378D544EA6B6DCEB7535B12980BB8A4D3 Jeffri H Frontz (jhfrontz)
 D3CC177286005BB8FF673294C5242A1AB3936517 jl2012
-82921A4B88FD454B7EB8CE3C796C4109063D4EAF Jon Atack
-32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli
+82921A4B88FD454B7EB8CE3C796C4109063D4EAF Jon Atack (jonatack)
+32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli (jonasschnelli)
 4B4E840451149DD7FB0D633477DFAB5C3108B9A8 Jorge Timon
```

Comment:
```suggestion
4B4E840451149DD7FB0D633477DFAB5C3108B9A8 Jorge Timon (jtimon)
```

Reply:
ACK, assuming he used the same key back then
```
$ git ls-files|grep -i /jtimon/
0.13.2-linux/jtimon/bitcoin-linux-0.13-build.assert
0.13.2-linux/jtimon/bitcoin-linux-0.13-build.assert.sig
0.13.2-win-unsigned/jtimon/bitcoin-win-0.13-build.assert
0.13.2-win-unsigned/jtimon/bitcoin-win-0.13-build.assert.sig
```